torch/csrc/autograd/python_variable.cpp

```diff
@@ -364,6 +364,54 @@ static PyObject* THPVariable_make_subclass(PyObject* _ignored, PyObject* args, P
   END_HANDLE_TH_ERRORS
 }
 
+static PyObject* THPVariable_make_wrapper_subclass(PyObject* _ignored, PyObject* args, PyObject* kwargs) {
+  HANDLE_TH_ERRORS
+  // NB: pin_memory doesn't actually do anything
+  // TODO: strides variant?
+  static PythonArgParser parser({
+    "_make_wrapper_subclass(PyObject* cls, IntArrayRef size, *, MemoryFormat? memory_format=None, ScalarType dtype=None, Layout layout=torch.strided, Device device=None, bool pin_memory=False, bool requires_grad=False)",
+  });
+  ParsedArgs<8> parsed_args{};
+  auto r = parser.parse(args, kwargs, parsed_args);
+  // TODO: handle has_torch_function, maybe?
```

Comment:
There is no Tensor input here right?
That would be for Mode?

Reply:
Yes. Also, if there's a hypothetical version of this which takes a Tensor and reads attributes directly from the Tensor maybe this should also be overrideable? I am not exactly sure what the use case for any of this is, it is mostly an argument from symmetry.

Reply:
I guess technically torch function isn't getting a mode so we can't override this anyway lol